pkg/scheduler/framework/plugins/podtopologyspread/scoring.go

```diff
@@ -256,3 +271,10 @@ func getPreScoreState(cycleState *framework.CycleState) (*preScoreState, error)
 	}
 	return s, nil
 }
+
+// topologyWeight calculates the weight for the topology.
+// Since topologySize is at least 1 (current node), and k8s supports 5k nodes,
+// result is in the interval <1.09, 8.52>
```

Comment:
perhaps I am missing something, but why is the minimum 1.09? what if size =1 (a single zone cluster), wouldn't that result in log(3) = 0.4?

Reply:
It's natural logarithm, not log10

Reply:
I remember something like that too, but it's not the case in C++, Python or go.